model-datastructure/src/commonMain/kotlin/org/modelix/model/lazy/PrefetchCache.kt

```diff
@@ -55,7 +56,17 @@ class PrefetchCache(private val store: IDeserializingKeyValueStore) : IDeseriali
         val missingHashes = hashes.filterNot { entries.containsKey(it) }
         val missingValues = store.getAll(missingHashes, deserializer).toList()
         val missingEntries = missingHashes.mapIndexed { index, hash -> hash to missingValues[index] }.associate { it }
-        return hashes.map { missingEntries[it] ?: entries[it] as T }
+        entries.putAll(missingEntries)
```

Comment:
@slisson Was my assumption correct, that `.getAll` should also cache the value like `.get` does?